src/stirling/source_connectors/socket_tracer/openssl_trace_bpf_test.cc

```diff
@@ -115,7 +121,7 @@ class BaseOpenSSLTraceTest : public SocketTraceBPFTestFixture</* TClientSideTrac
   }
 
   TServerContainer server_;
-  bool force_fptr_ = TForceFptrs;
+  bool use_new_tls_impl_ = UseNestedSyscallFD;
```

Comment:
It may be possible to make this `static constexpr` and/or you could just use the template parameter directly (i.e. there may be no need to create a "local" copy of the template parameter).

Reply:
I went for using the template parameter directly. Updated in latest revision.